File: reservation.py
class Reservation:
    def __init__(self, id, location, number_of_rental_days):
        self.id = id
        self.location = location
        self.number_of_rental_days = number_of_rental_days

    def get_id(self):
        return self.id

    def get_location(self):
        return self.location

    def get_number_of_rental_days(self):
        return self.number_of_rental_days

File: src/main/java/com/prog5/loc/Reservation.java
package com.prog5.loc;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class Reservation {
    private long id;
    private location location;
    private int numberOfRentalDays;
}
